src/Tgstation.Server.Host.Console/tgs.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=tgstation-server
+After=network.target
+After=mariadb.target
+
+[Service]
```

Comment:
Should have a [type](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=) definition.

Reply:
Also, is the recommendation really to run this as root? That seems terrible from a best practices perspective.

Reply:
Pretty sure it's needed for process priority elevation stuff, plus some of the event scripts install stuff.

Also the windows service runs as LOCAL SERVICE since you need admin rights to even make symlinks on windows.

Reply:
> Should have a [type](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=) definition.

simple is the default type, and the type we want to run it under since the initial script uses exec instead of spawn

Reply:
> > Should have a [type](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=) definition.
> 
> simple is the default type, and the type we want to run it under since the initial script uses exec instead of spawn

You should always be explicit.

Reply:
> Also the windows service runs as LOCAL SERVICE since you need admin rights to even make symlinks on windows.

LOCAL SERVICE is, as you might gleam from the name, a service account and maintains minimum privileges. It's not a root-adjacent account. 

> Pretty sure it's needed for process priority elevation stuff

That should be handled in a limits.conf file. Alternatively, the user could be given privileges to edit the niceness values of their processes.

> plus some of the event scripts install stuff.

That should not be the default if you're running bare metal. Best practice should be to install everything you need before running TGS as a system service.

Reply:
> 
> Correction, its `LOCAL SYSTEM`
> 
> MSFT docs: `An account, used by the service control manager, that has extensive privileges on the local computer and acts as the computer on the network.`

LOCAL SYSTEM **is** a root-adjacent role. Although, it's still restricted in that it only has access to local and domain resources. It can't access external resources, which is not a restriction root has on Linux.

Reply:
Reminder that 
`AmbientCapabilities=CAP_SYS_NICE`

AFAIK, that covers all the things ***TGS*** needs from running as root.

Note that sample EventScripts for tgstation are built with the expectation of running as root